topi/python/topi/cuda/nms.py

```diff
@@ -182,8 +182,15 @@ def calculate_overlap(out_tensor, box_a_idx, box_b_idx):
 
 
 @non_max_suppression.register(["cuda", "gpu"])
-def nms_gpu(data, valid_count, return_indices, iou_threshold=0.5, force_suppress=False,
-            topk=-1, id_index=0, invalid_to_bottom=False):
+def nms_gpu(data,
+            valid_count,
+            max_output_size=-1,
+            iou_threshold=0.5,
+            force_suppress=False,
+            topk=-1,
```

Comment:
top_k

Reply:
renamed 'topk' to `top_k` inside `nms_gpu` method